src/pygeoprocessing/routing/routing.pyx

```diff
@@ -93,33 +93,33 @@ cdef int _MAX_PIXEL_FILL_COUNT = 500
 cdef extern from "<queue>" namespace "std" nogil:
     cdef cppclass priority_queue[T, Container, Compare]:
         priority_queue() except +
-        priority_queue(priority_queue&) except +
-        priority_queue(Container&)
+        priority_queue(priority_queue & ) except +
+        priority_queue(Container & )
         bint empty()
         void pop()
-        void push(T&)
+        void push(T & )
         size_t size()
-        T& top()
+        T & top()
 
 # this is a least recently used cache written in C++ in an external file,
 # exposing here so _ManagedRaster can use it
 cdef extern from "LRUCache.h" nogil:
     cdef cppclass LRUCache[KEY_T, VAL_T]:
         LRUCache(int)
-        void put(KEY_T&, VAL_T&, clist[pair[KEY_T,VAL_T]]&)
-        clist[pair[KEY_T,VAL_T]].iterator begin()
-        clist[pair[KEY_T,VAL_T]].iterator end()
+        void put(KEY_T &, VAL_T&, clist[pair[KEY_T, VAL_T]]&)
+        clist[pair[KEY_T, VAL_T]].iterator begin()
+        clist[pair[KEY_T, VAL_T]].iterator end()
         bint exist(KEY_T &)
         VAL_T get(KEY_T &)
-        void clean(clist[pair[KEY_T,VAL_T]]&, int n_items)
+        void clean(clist[pair[KEY_T, VAL_T]] &, int n_items)

```

Comment:
Okay I'm going to stop commenting on this because it's everywhere. I'll put it in the main review comment.